### DISCUSSION TITLE
Audacity's theme colors aren't correct on certain UI elements

### DISCUSSION THREAD

**Discussion by Delfite** [audacity/audacity#6652] [General]
### Bug description

Audacity, from what I've read online, is supposed to use your system's currently active theme for some of it's UI elements, such as the scrollbars. The issue with this is that Audacity just doesn't seem to do this on Windows 10 22H2, nor do the UI elements attempt to match the colors of the theme you have selected in Audacity's settings. Examples are as follows:

Despite me setting my Windows theme to **Dark**...
![image](https://github.com/audacity/audacity/assets/23504262/56001476-ebef-4ba7-93cc-cae9ee5bccb1)

Audacity still thinks it's **Light**. How unsightly!
![image](https://github.com/audacity/audacity/assets/23504262/f96b2e2d-f6ce-47e1-8c59-61f84934e8ab)

### Steps to reproduce

1. Open Audacity
2. Navigate to **Edit > Preferences > Interface** and switch the theme to either Dark mode or High Contrast mode
3. Set your Windows theme to Dark Mode through your system settings

### Expected behavior

Certain UI elements should be using my system's active theme OR use the same colors as the theme currently selected in Audacity's settings.

### Actual behavior

Certain UI elements aren't using my system's active theme NOR are they using the colors of the theme currently selected in Audacity's settings.

### Audacity Version

latest stable version (from audacityteam.org/download)

### Operating system

Windows

**Comment by LWinterberg** [comment]
There's different versions of windows in Windows, with some windows being capable of knowing and adapting to the theme and others which can't. In Windows 11, tools like regedit and device manager, or even the item properties dialog (Alt+Enter) of the windows explorer aren't themed either.

**Reply by Delfite** [reply]
Not trying to sound needy or impatient when I ask this but, is this issue getting fixed at some point? The issue was closed as completed, but as of 3.6.1 the UI still has a lot of the same problems as before:

The window's title bar is still white when it should actually be black.
The scrollbars are still white when they should actually be a dark gray.
The borders and backgrounds of certain boxes are still white when they should actually be something darker.

![image](https://github.com/user-attachments/assets/768bbdcf-8476-48c8-8df7-4ba7bd2347fd)

**Reply by ElTioRata** [reply]
Still present in 3.7.5

**Reply by Norumu** [reply]
This is still happening in 3.7.8
